src/lib/components/forms/ClassDetailsForm.svelte

```diff
@@ -149,6 +161,8 @@
             new Date(semesterDates.classesEnd),
           )
           values.meetingTimes = meetingTimes
+          values.feedbackCompleted = new Array(meetingTimes.length).fill(false)
+          values.classesStatus = new Array(meetingTimes.length).fill('sometime')
```

Comment:
use the enum we have, check the whole codebase for instances of this. BTW, i renamed the enum, so make sure that doesnt cause issues